README.md

```diff
@@ -21,9 +21,14 @@ deploy_pins is exactly what you need.
 
 To generate new task template file
 ```bash
-rails g deploy_pin:task
+rails g deploy_pin:task some_task_title
```

Comment:
make sure the title is mandatory, please

Reply:
<img width="851" alt="Screenshot 2022-08-26 at 02 40 11" src="https://user-images.githubusercontent.com/32734554/186787419-46a71d19-c817-4c0d-a0f4-5a7197db8a9a.png">
it is